hive/build.gradle

```diff
@@ -5,16 +5,38 @@ description = "Elasticsearch Hadoop Hive"
 
 evaluationDependsOn(':elasticsearch-hadoop-mr')
 
+configurations {
+    embedded {
+        transitive = false
+        canBeResolved = true
+    }
+    implementation {
+        extendsFrom project.configurations.embedded
```

Comment:
I don't think we should change the `canBeResolved` for the implementation configuration. See https://discuss.gradle.org/t/what-is-a-configuration-which-cant-be-directly-resolved/30721/2 
> In short, you should never change the canBeConsumed and canBeResolved flags of configuration you do not create.

also, it's not clear you are changing it ?

Reply:
Implementation shouldn't be affected by that at all. Only the `embedded` configuration is getting `canBeResolved` set on it. The `implementation` configuration just depends on it

Reply:
ok , i mis-understood how `extendsFrom` behaves